services/galley/schema/src/V77_MLSSubconversation.hs

```diff
@@ -0,0 +1,36 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V77_MLSSubconversation (migration) where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 77 "Add a new version member_client that is indexed by group_id" $ do
+    schema'
+      [r| CREATE TABLE member_client_v2 (
```

Comment:
How about naming the table with something more descriptive, e.g., `member_client_by_groupid`? "v2" can be confused with lots of things, including the client API v2.

Reply:
I rename it to `mls_group_member_client`. Motivation for the `v2` suffix was so that it is listed next to the table is deprecating (for better discoverability). Found a different solutoin: Add the note when generating the schema dump